cpp/libsarmata-client/RemoteSession.cpp

```diff
@@ -22,6 +25,7 @@ void RemoteSession::Open(const std::string & token, const ASRSessionSettings & s
 {
     stub_ = ASR::NewStub(grpc::CreateChannel(host_, grpc::InsecureChannelCredentials()));
     stream_ = stub_->Recognize(&context_);
+	samplesStreamCompleted = false;
```

Comment:
When we moved initialization to the ctor, we assume that Open() could be called just once. And this is not asserted here. Is it correct assumption?